docs/source/teams/pluggable_auth.md

```diff
@@ -124,59 +120,60 @@ API to ensure a comprehensive transfer of information.
 For each user, the migration involves several steps:
 
 1. Creating a FiftyOne Teams user profile via `POST /cas/api/users`.
-2. Assigning the user to the default organization through a membership entry.
-3. Linking the user's account to the new authentication provider by creating an
+1. Assigning the user to the default organization through a membership entry.
+1. Linking the user's account to the new authentication provider by creating an
    account reference via `POST /cas/api/accounts`.
 
 The final step in the migration involves changing the `FIFTYONE_AUTH_MODE`
 setting from legacy to internal. This change officially activates Internal
 Mode, completing the migration process.
 -->
 
-## Getting started with internal mode
+## Getting Started with Internal Mode
```

Comment:
The style for our docs is like this:

> Getting started with internal mode

Here's a few examples from a random search:

 - `Selecting samples of interest`
 - `Add predictions to the dataset`
 - `Removing redundant images`

Reply:
Seems like we inconsistently capitalize headings. I'll change it back as you request.